### PR TITLE
Development nginx default page

### PR DESCRIPTION
**Pull request by RafyAmgadBenjamin**
### Description

Update our Nginx configuration to load our default HTML page
### Changes
- added `sals/nginx/static` directory 
- Updated `nginx.conf` template
- Updated `sals/nginx.pu`

### Related Issues
https://github.com/threefoldtech/js-sdk/issues/505

### CODE REVIEW

File: jumpscale/sals/nginx/static/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    font-size: 300%;
+    text-align: center;
+}
+}
+
+</style>
+</head>
+<body>
+    <div class="h-center">
+        <p class="large-font">welcome to 3Bot go to your admin dashboard from here</p>
```
Comment (waleedhammam) [inline jumpscale/sals/nginx/static/index.html:63]: Can we adjust this typo a little starts with capital letter

```suggestion
        <p class="large-font">
            Welcome to 3Bot
        </p>
```

File: jumpscale/sals/nginx/static/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+
+    <div class="center">
+        <h1>Go to /admin</h1>
```
Comment (waleedhammam) [inline jumpscale/sals/nginx/static/index.html:67]: and modify the header, but i think css should be modified to correspond ok ?

```suggestion
        <h3>Press the following button to access your admin dashboard</h3>
```

**Review by waleedhammam — CHANGES_REQUESTED** [review]

File: jumpscale/sals/nginx/static/index.html (hunk 1 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/sals/nginx/static/index.html:63]: done

File: jumpscale/sals/nginx/static/index.html (hunk 2 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/sals/nginx/static/index.html:67]: done

**Comment by waleedhammam** [comment]
Tested against development_capacitypool

start --local
install marketplace package
go to https://localhost:8443/marketplace
you will find 404

**Comment by abom** [comment]
@waleedhammam It's the config of marketplace:

```
name = "marketplace"
ports = [ 80, 443]
```

That's why it does not work on 8443.

**Review by waleedhammam — APPROVED** [review]

**Review by BolaNasr — APPROVED** [review]

**Review by abom — APPROVED** [review]

**Pull request merged**